### DISCUSSION TITLE
Clarification on iframe usage

### DISCUSSION THREAD

**Discussion by PlkMarudny** [bluwy/svelte-fast-dimension#3] [Q&A] [answered]
Do I get it correctly - after using the package, iframes should disappear from the generated code?

**Comment by bluwy** [comment, marked as answer]
Yes, `iframe` should now not be used by Svelte after adding the preprocessor. Technically the generated code wouldn't have `iframe` either ways since it's something injected in runtime with [this code](https://github.com/sveltejs/svelte/blob/fc4797c6f84cf299117243e7193bf77d96914e85/src/runtime/internal/dom.ts#L585-L627).
